cep-6.md

```diff
@@ -0,0 +1,233 @@
+also metadata about the message, including information about how often the client should display the message (more on 
+this in the next section).
+
+Here's an example of the `notices.json` file which will be stored in the root of the channel directory structure.
+
+```json
+{
+  "notices": [
+    {
+      "id": "1cd1d8e5-d96c-42d1-9c29-e8120ad80823",
+      "message": "Here is an example message that will be displayed to users",
+      "level": "info",
+      "created_at": "2022-04-26T11:50:34+00:00",
+      "expiry": 604800,
+      "interval": 604800
+    }
+  ]
+}
+```
+
+Detailed overview of the JSON fields:
+
+- **notices** [Array] holds zero or more notices that will be displayed to the client.
+  - **id** [String] unique ID for the message itself. UUIDs are preferred, but there is no required format.
+  - **message** [String] message that gets displayed to users.
+  - **level** [String] one of (info|warning|critical). These will let our users know the category of the message
+    and will also allow the client to apply different formatting rules (e.g. text color).
+  - **created_at** [String] ISO 8601 formatted timestamp showing the creation time of the message.
+  - **expiry** [Number] starting at `created_at`, a number specifying how long in seconds the message is valid for.
+  - **interval** [Number] starting from the modified time of a cache file, the time in seconds specifying how often
+    this is shown to the user.
+  - **arch** Optional[String] allows messages to be targeted to users of a certain architecture.
+    - Example value: `win-64`, `linux-64`, etc.
+  - **conda_version** Optional[String] allows messages to be targeted to users of a certain `conda` version.
+    - Example value: `4.12.0`
+
+### How often will these messages appear?
+
+How often the messages appear will be configurable by the channel owners and the client. This will be accomplished by 
+the expiry field in the `notices.json` file itself, but the client will the have ultimate say over whether this
+message is displayed. We will provide clients with a setting to permanently disable these messages in their `.condarc`
+files:
+
+```yaml
+display_notices: false
+```
+
+## Motivation
+
+The motivation for this CEP came about as channel owners (Anaconda specifically) needed a way to broadcast messages to
+users of their channels. These messages may contain specific notices for particular users (e.g. identified by
+IP address) or general messages for the wider audience of a particular channel.
+
+Additionally, this new notification space can also provide a place for us to relocate `conda update conda` reminders 
+to a more visible spot (at the end of command output versus in the middle of the output). On top of this, other channels
+can use these notices as a way to share news with their users or requests for help in maintaining their channels.
+
+
+## Rationale
+
+TBD
```

Comment:
It'd be good to fill in reasons why this was implemented in a client/server model or something like this. If there are no added details for design decisions you can also just skip it, of course.